### ISSUE TITLE
Error when Copy and Pasting from Text Wrangler

### ISSUE THREAD

**Issue by yannickmcosta** [careteditor/caret#242]
The following error has occurred:
```
{
    "message": "Cannot read property 'type' of undefined",
    "version": "1.14.0",
    "os": "macOS Sierra",
    "path": "web/Editor/Editor.Markdown.js",
    "code": "_0x919fxf[_0x9688[120]][_0x9688[205]](_0x919fx2)})",
    "line": 5,
    "column": 42160,
    "stack": [
        "at /Applications/Caret.app/Contents/Resources/app.asar/web/Editor/Editor.Markdown.js:5:42160",
        "at Array.find ((native)",
        "at _0x919fx2.value (web/Editor/Editor.Markdown.js:5:42123)",
        "at _0x2bbcx3.value (web/Events.js:5:1099)",
        "at _0x317bx2.<anonymous> (web/Editor/Editor.Input.js:5:18414)",
        "at t.value (web/Editor/Editor.Selection.js:5:1265)",
        "at _0x2bbcx3.value (web/Editor/Editor.js:5:12324)",
        "at _0x317bx2.value (web/Editor/Editor.Input.js:5:18238)",
        "at _0x317bx2.value (web/Editor/Editor.Input.js:5:21405)"
    ]
}
```

**Comment by astoilkov** [comment]
We fixed the issue internally. The fix will be included in the next version release. I will write in this thread when the release is publicly available.

Thanks for reporting.

**Comment by yannickmcosta** [comment]
👍  Great stuff, look forward to it, thanks @astoilkov

**Comment by astoilkov** [comment]
We released Caret 1.14.1 where a fix for this bug is now included.

Closing this issue.
